rust/chains/tw_ton/src/transaction.rs

```diff
@@ -20,6 +21,7 @@ pub struct SignedTransaction {
     /// Created via `StateInit`.
     pub state_init: Option<StateInit>,
     pub signed_body: SignedMessage,
+    pub wallet_version: WalletVersion,
```

Comment:
Here can be a Cell instead of `SignedMessage`, so there is no need to keep `wallet_version` here